lib/utils.js

```diff
@@ -435,3 +438,39 @@ function prettifyError ({ keyName, lines, eol, ident }) {
 
   return result
 }
+
+/**
+ * Deletes a specified property from a log object if it exists
+ *
+ * @param {object} log The log object to be modified.
+ * @param {string} property A string identifying the property to be deleted from
+ * the log object. Accepts nested properties delimited by a `.`
+ * e.g. `'prop1.prop2'`.
+ */
+function deleteLogProperty (log, property) {
```

Comment:
Please add tests to https://github.com/pinojs/pino-pretty/blob/d9fc678fb546c466e9df02cd8eb4da35a8c26e7e/test/lib/utils.internals.test.js

Reply:
Updated in the latest commit